storage/preferences/src/main/java/de/danoeh/antennapod/storage/preferences/UserPreferences.java

```diff
@@ -530,10 +532,15 @@ public static int getEpisodeCacheSize() {
         return Integer.parseInt(prefs.getString(PREF_EPISODE_CACHE_SIZE, "20"));
     }
 
+    @Deprecated
```

Comment:
I would remove this instead of deprecating. You can use PreferenceUpgrader to migrate to the new preference key. Then future developers cannot accidentally use the old value

Reply:
Yeah, that is still a leftover. I wanted to get a mvp out before i started fixing tests and all those occurences.